### DISCUSSION TITLE
Qtable in grid mode in fullscreen broken!!!

### DISCUSSION THREAD

**Discussion by davidreyg** [quasarframework/quasar#12607] [General - Components / Directives / etc]
### What happened?

I try to toggle fullscreen mode in grid mode 
![qtable](https://user-images.githubusercontent.com/19297294/154769392-b4cdd288-6bc9-429f-ba80-33b49c0c9569.png)
But they background is like transparent
BTW in normal style it works
![fullscreen grid](https://user-images.githubusercontent.com/19297294/154769722-d0922b22-a089-4d1b-ab02-7eb468e0c8d6.png)

Ando in Docs i didnt find anything related to this :(.

### What did you expect to happen?

![fullscreen grid with background wihte!](https://user-images.githubusercontent.com/19297294/154769548-e231520f-9408-420f-b1d3-f2df398568f6.png)

**Comment by MilosPaunovic** [comment]
On this docs example everything works as expected:
[https://quasar.dev/vue-components/table#visible-columns-custom-top-fullscreen](https://quasar.dev/vue-components/table#visible-columns-custom-top-fullscreen)

As you can see here as well:
[https://codepen.io/milospaunovic/pen/mdqGaRv?editors=1010](https://codepen.io/milospaunovic/pen/mdqGaRv?editors=1010)

Could you try to explain your problem a little bit more, as it seems like it has something to do with your layout rather than the table itself.

**Reply by davidreyg** [reply]
[Index.zip](https://github.com/quasarframework/quasar/files/8147689/Index.zip)
Hi @MilosPaunovic thanks for answering
I tried to Reproduce your code using the default layout from quasar LayoutBuilder
Still have the error in the picture

**Reply by davidreyg** [reply]
https://codepen.io/davidreyg/pen/WNXgmvK

**Comment by MilosPaunovic** [comment]
You were completely right - there was an issue with table background when using grid mode along with fullscreen.

I've submitted PR #12614 which resolves that issue - maybe the guys from the core team will find a better solution but it will be handled soon.

**Reply by davidreyg** [reply]
Hey @MilosPaunovic thank you so much for your help
